Dockerfile

```diff
@@ -1,17 +1,21 @@
-FROM messense/rust-musl-cross:x86_64-musl AS builder
+FROM messense/rust-musl-cross:x86_64-musl AS builder-amd64
+FROM messense/rust-musl-cross:aarch64-musl AS builder-arm64
+
+FROM builder-${TARGETARCH} AS builder
```

Comment:
```suggestion
# hadolint ignore=DL3006
FROM builder-${TARGETARCH} AS builder
```
To skip this [warning](https://github.com/dotenv-linter/dotenv-linter/actions/runs/8126319385/job/22758530295).